<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patientDatabase</class>
 <widget class="QDialog" name="patientDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>HOME</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_images">
           <property name="text">
            <string>ADD NEW IMAGES TO PATIENT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/new/Images/button.png);
 color: black;
            
            border-top: 3px transparent;
            border-bottom: 3px transparent;
            border-right: 10px transparent;
            border-left: 10px transparent;</string>
           </property>
           <property name="text">
            <string>UPDATE INFO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/new/Images/187850234.jpg);
 color: black;
            
            border-top: 3px transparent;
            border-bottom: 3px transparent;
            border-right: 10px transparent;
            border-left: 10px transparent;</string>
           </property>
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTableView" name="tableView_view">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Patient Information</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>PatientId</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_patientId</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_patientId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>First</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_first</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_first">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Middle</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_middle</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_middle"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Last</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_last</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_last"/>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Suffix</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_suffix</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_suffix"/>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Date of Birth</string>
            </property>
            <property name="buddy">
             <cstring>dateEdit_dob</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_dob">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Gender</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_gender</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_gender">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Male</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Female</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Anatomy</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_anatomy</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_anatomy">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hand</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Elbow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shoulder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Foot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ankle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Knee</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Upper Leg</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="16" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Physician</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_physician</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_physician"/>
          </item>
         </layout>
        </item>
        <item row="17" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_Description</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Description">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="18" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Accession</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_Accession</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Accession"/>
          </item>
         </layout>
        </item>
        <item row="18" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Date and Time</string>
            </property>
            <property name="buddy">
             <cstring>dateTimeEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Prefix</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEdit_prefix"/>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
       <zorder>label_13</zorder>
       <zorder>lineEdit_prefix</zorder>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_patientId</tabstop>
  <tabstop>lineEdit_first</tabstop>
  <tabstop>lineEdit_middle</tabstop>
  <tabstop>lineEdit_last</tabstop>
  <tabstop>lineEdit_suffix</tabstop>
  <tabstop>comboBox_gender</tabstop>
  <tabstop>dateEdit_dob</tabstop>
  <tabstop>comboBox_anatomy</tabstop>
  <tabstop>lineEdit_physician</tabstop>
  <tabstop>lineEdit_Description</tabstop>
  <tabstop>lineEdit_Accession</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>pushButton_edit</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>tableView_view</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
